docs/aws-config.md

```diff
@@ -298,7 +276,7 @@ It can be set up for Velero by creating a role that will have required permissio
       --policy-name velero-policy \
       --policy-document file://./velero-policy.json
     ```
-4. Update `AWS_ACCOUNT_ID` & `VELERO_ROLE_NAME` in the file `config/aws/10-deployment-kube2iam.yaml`:
+4. Update `AWS_ACCOUNT_ID` & `VELERO_ROLE_NAME` with `kubectl edit deploy/velero -n velero` and add the following annotation:
```

Comment:
should we explicitly have the user run `velero install --dry-run -o yaml`, then make this edit there, then apply the YAML?

Reply:
That may work, though using `kubectl apply` doesn't have the CRD wait that our command and Helm have, so I'm reticent to say to use it right now.

Reply:
The other problem here - `--secret-file` is currently required.  Maybe we need some way of telling Velero that some sort of IAM framework is being used, so leaving off the secret is fine?

Reply:
ah, hmm, guess this is a little tricky then. is the answer "use Helm"?

Reply:
I think so, at least for now. We need to get some Helm install instructions in, but I'd rather do that in a separate, dedicated PR.

Reply:
yeah that works.